index.html

```diff
@@ -244,7 +244,7 @@ <h4 class="con-us">CONTACT US</h4>
                 </li>
                 <li class="li-item">
                   <label for="textarea">
-                    <textarea name="text" id="text"  class="text-input detail-input" placeholder="Enter text here..." cols="23" rows="5" max-length="500" required></textarea>
+                    <textarea class="text-input detail-input" name="text" id="text" cols="23" rows="5" placeholder="Enter text here..." max-length="500" required></textarea>
                   </label>
                 </li>
                 <li class="btn-container">
```

Comment:
- [ ] I kindly suggest that you add the different button interactions including active (when it's pressed).

![image](https://user-images.githubusercontent.com/52934128/178548917-051dc42c-5bf6-4a46-b635-87e4a6b96a77.png)

the button on the pressed state has the `background: #5E42D7;`
### EXAMPLE:
**for pressed state**
```
button:active {
   background-color: #5E42D7;
}
```